### PR TITLE
Add support for volumes blacklisting (fixes #21)

### LINKED ISSUE #21: Allow to blacklist some volumes with environment variable

### PR DESCRIPTION
**Pull request by mcanevet**

### CODE REVIEW

File: handler/handler.go [hunk 1]
```diff
@@ -23,6 +23,7 @@ type environment struct {
 	SwiftRegionName    string `env:"SWIFT_REGIONNAME"`
 	FullIfOlderThan    string `env:"FULL_IF_OLDER_THAN" envDefault:"15D"`
 	RemoveOlderThan    string `env:"REMOVE_OLDER_THAN" envDefault:"30D"`
+	VolumesBlacklist   string `env:"CONPLICITY_VOLUMES_BLACKLIST"`
```
Comment (raphink) [inline handler/handler.go]: Why not

```go
VolumesBlacklist   []string `env:"CONPLICITY_VOLUMES_BLACKLIST"`
```

File: conplicity.go [hunk 2]
```diff
@@ -42,6 +43,14 @@ func backupVolume(c *handler.Conplicity, vol *docker.Volume) (err error) {
 		return
 	}
 
+	list := c.VolumesBlacklist
+	sort.Strings(list)
```
Comment (raphink) [inline conplicity.go]: How about sorting this in `getEnv()` so it's done once and for all?

File: conplicity.go [hunk 3]
```diff
@@ -42,6 +43,13 @@ func backupVolume(c *handler.Conplicity, vol *docker.Volume) (err error) {
 		return
 	}
 
+	list := c.VolumesBlacklist
+	i := sort.SearchStrings(list, vol.Name)
+	if i < len(list) && list[i] == vol.Name {
+		log.Infof("Ignoring blacklisted volume " + vol.Name)
+		return
+	}
+
```
Comment (raphink) [inline conplicity.go]: Add this to documentation in README? :smile:

Comment (mcanevet) [inline conplicity.go]: done

**Pull request merged**